### PR TITLE
Adding Credits Screen

### PR DESCRIPTION
**Pull request by morganick**
# Description

We wanted to make sure we recognized the all the folks who made this conference and app possible.

[Trello Card](161-credits) #127 

This creates another screen under the Info tab called "Credits".

# Graphics

| iOS            | Android            |
| -------------- | ------------------ |
| image | image |
| image | image |

# Checklist:

- [x] I have done a thorough self-review of my code
- [ ] I have tested with a screen reader and font-scaling turned on and added necessary accessibility features
- [x] I have run tests and linter

### CODE REVIEW

File: app/i18n/en.ts [hunk 1]
```diff
@@ -85,6 +85,8 @@ const en = {
     goldSponsor: "Gold Sponsor",
     silverSponsor: "Silver Sponsors",
     bronzeSponsor: "Bronze Sponsors",
+    creditsTitle: "Credits",
+    creditsHeading: "Thank you to everyone who worked on the app this year!",
```
[the remark this answers is not in the document]
Comment (morganick) [reply app/i18n/en.ts:89]: @GantMan suggestions?

[the remark this answers is not in the document]
Comment (GantMan) [reply app/i18n/en.ts:89]: For a header <h1> Style I like it short.

Something like "Special Thanks To" is general and each name has the associated credit on why they are there.

If you want to say more like you're thinking, you need an <h2> style for the second part.

I'd go with fewer words.  But if you want an h1/h2 lead in, that's cool, too.  We just need to see it balanced.

File: app/screens/InfoScreen/CreditsScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+      name: "Jamon Holmgren",
+      role: "Co-organizer of Chain React & App Development",
+    },
+    {
+      name: "Jenna Fucci",
+      role: "Designer (web, app, and print)",
+    },
+    {
+      name: "Simran Sachdeva",
+      role: "CFP Committee & Workshop Assistant",
```
Comment (mazenchami) [inline app/screens/InfoScreen/CreditsScreen.tsx:43]: she's also a tester

File: app/i18n/en.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (jamonholmgren) [reply app/i18n/en.ts:89]: Throw it into ChatGPT 4 and ask it to shorten it or make it a subtitled title

File: app/screens/InfoScreen/CreditsScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+        <FlashList
+          ListHeaderComponent={
+            <Text preset="screenHeading" tx="infoScreen.creditsHeading" style={$heading} />
+          }
+          data={data}
+          renderItem={({ item }) => {
+            return (
+              <View style={$person}>
+                <Text preset="companionHeading" text={item.name} />
+                <Text preset="label" style={$role} text={item.role} />
```
[the remark this answers is not in the document]
Comment (justinhuskey) [reply app/screens/InfoScreen/CreditsScreen.tsx:123]: I like this @morganick! Nice

**Comment by robinheinze** [comment]
I love that we're adding this ❤️ Great job @morganick!

File: app/screens/InfoScreen/CreditsScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+      role: "App Testing",
+    },
+    {
+      name: "Jed Bartausky",
+      role: "App Testing",
+    },
+    {
+      name: "Ellie Croce",
+      role: "App Testing",
+    },
```
[the remark this answers is not in the document]
Comment (morganick) [reply app/screens/InfoScreen/CreditsScreen.tsx]: Updated in [0caad88](https://github.com/infinitered/ChainReactApp2023/pull/154/commits/0caad88a06c41933d07e95091cfc1c9ca3c3b5cc)

File: app/screens/InfoScreen/CreditsScreen.tsx [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+      name: "Mazen Chami",
+      role: "App Development",
+    },
+    {
+      name: "Frank Calise",
+      role: "App Development",
+    },
+    {
+      name: "Leon Kim",
+      role: "App Development",
```
[the remark this answers is not in the document]
Comment (morganick) [reply app/screens/InfoScreen/CreditsScreen.tsx]: Updated in [0caad88](https://github.com/infinitered/ChainReactApp2023/pull/154/commits/0caad88a06c41933d07e95091cfc1c9ca3c3b5cc)

File: app/screens/InfoScreen/CreditsScreen.tsx (hunk 2 above)
Comment (morganick) [reply app/screens/InfoScreen/CreditsScreen.tsx]: Updated in [0caad88](https://github.com/infinitered/ChainReactApp2023/pull/154/commits/0caad88a06c41933d07e95091cfc1c9ca3c3b5cc)

File: app/screens/InfoScreen/CreditsScreen.tsx [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+    leftIcon: "back",
+    onLeftPress: () => navigation.goBack(),
+  })
+
+  type Person = {
+    name: string
+    role: string
+  }
+
+  const data: Person[] = [
```
[the remark this answers is not in the document]
Comment (morganick) [reply app/screens/InfoScreen/CreditsScreen.tsx]: Updated in [0caad88](https://github.com/infinitered/ChainReactApp2023/pull/154/commits/0caad88a06c41933d07e95091cfc1c9ca3c3b5cc)

File: app/i18n/en.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (morganick) [reply app/i18n/en.ts]: ChatGPT let me down. Going with @GantMan's answer.

**Review by mazenchami — APPROVED** [review]
LGTM! 🎉 🚀

**Pull request merged**